### DISCUSSION TITLE
data_table_formatting.py is not running with bokeh 3.3.4

### DISCUSSION THREAD

**Discussion by GillesFa** [bokeh/bokeh#13733] [General]
Hi All

I copied the example in https://github.com/bokeh/bokeh/blob/branch-3.4/examples/interaction/widgets/data_table_formatting.py in my local laptop. I did not change anything but when I run it I get the following error:

> Exception has occurred: ValueError
> failed to validate DateFormatter(id='p1005', ...).font_style: invalid value: 'weekend_bold'; allowed values are normal, italic, bold or bold italic
>   File "C:\Development\Net-Zero_Energy_Networks\energy_optimisation\scratch\bokeh_data_table1.py", line 32, in <module>
>     formatter=DateFormatter(
>               ^^^^^^^^^^^^^^
> ValueError: failed to validate DateFormatter(id='p1005', ...).font_style: invalid value: 'weekend_bold'; allowed values are normal, italic, bold or bold italic

The error seems to be related to this part of the code when setting the "font_style" of the DateFormatter:

```
table = DataTable(
    source=ColumnDataSource(data),
    columns=[
        TableColumn(
            field="dates",
            title="Date",
            formatter=DateFormatter(
                format="%A, %b %-d, %Y",
                font_style="weekend_bold",
                background_color=factor_cmap(
                    field_name="is_weekend",
                    palette=["lightgrey", "white"],
                    factors=["True", "False"],
                ),
            ),
        ),

```

Please can you provide a fix?
thanks
Gilles

**Comment by mattpap** [comment]
This example requires bokeh 3.4 (currently 3.4.0rc1).

**Comment by GillesFa** [comment]
When will bokeh 3.4 available using pip?

**Reply by mattpap** [reply]
Probably early next week. rc1 is available on pypi.

**Reply by GillesFa** [reply]
Thanks
